IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/it-IT/Resources.resw

```diff
@@ -635,11 +635,11 @@
     <comment>Format string for message displayed when Inbound pairing handler registration already done</comment>
   </data>
   <data name="BluetoothInboundRegistrationFailed" xml:space="preserve">
-    <value>Registrazione per l'associazione in entrata non riuscita</value>
+    <value>Registrazione per l'associazione in entrata non riuscita.</value>
     <comment>Message displayed when Inbound pairing handler registration fails</comment>
   </data>
   <data name="BluetoothInboundRegistrationSucceededFormat" xml:space="preserve">
-    <value>Registrazione per l'associazione in entrata completata. Procedure supportate: {0}</value>
+    <value>Registrazione per l'associazione in entrata completata. Procedure supportate: {0}. </value>
```

Comment:
String in English: Successfully registered for inbound pairing, supported ceremonies: {0} . 
Note that the (,) is replaced by a (.)